### PR TITLE
load data: update the notes about committing rows

### PR DESCRIPTION
**Pull request by qiancai**
### What is changed, added or deleted? (Required)

Add the behavior description about committing rows for v7.0.0 and later versions.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.4 (TiDB 7.4 versions)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by ekexium — APPROVED** [review]

File: sql-statements/sql-statement-load-data.md [hunk 1]
```diff
@@ -117,9 +117,10 @@ TiDB 中的 `LOAD DATA` 语句语法上兼容 MySQL（除字符集选项被解
 
 > **注意：**
 >
-> - 在 TiDB v4.0.0 之前的版本中，`LOAD DATA` 语句每 20000 行进行一次提交。
-> - 从 TiDB v4.0.0 开始一直到 TiDB v6.6.0 的版本，TiDB 默认在一个事务中提交所有行。
-> - 从 TiDB v4.0.0 及以前版本升级后，可能出现 `ERROR 8004 (HY000) at line 1: Transaction is too large, size: 100000058` 错误。要解决该问题，建议调大 `tidb.toml` 文件中的 [`txn-total-size-limit`](/tidb-configuration-file.md#txn-total-size-limit) 值。如果无法增加此限制，还可以将 [`tidb_dml_batch_size`](/system-variables.md#tidb_dml_batch_size) 的值设置为 `20000` 来恢复升级前的行为。注意自 v7.0.0 起，`tidb_dml_batch_size` 对 `LOAD DATA` 语句不再生效。
+> - 在 TiDB v4.0.0 之前的版本中，`LOAD DATA` 语句每 20000 行进行一次提交。该行数不支持更改。
+> - 从 TiDB v4.0.0 开始一直到 TiDB v6.6.0 的版本，TiDB 默认在一个事务中提交所有行。如需 `LOAD DATA` 语句按照每固定的行数进行一次提交，可以设置 [`tidb_dml_batch_size`](/system-variables.md#tidb_dml_batch_size) 为所需的行数。
+> - 从 v7.0.0 起，TiDB 默认在一个事务中提交所有行，`tidb_dml_batch_size` 对 `LOAD DATA` 语句不再生效。
```
Comment (D3Hunter) [inline sql-statements/sql-statement-load-data.md:122]: cc @lance6716

Comment (lance6716) [reply sql-statements/sql-statement-load-data.md:122]: 在 7.0.0 版本中，`tidb_dml_batch_size` 确实不再控制 LOAD DATA 语句。新增了 LOAD DATA ... WITH batch_size = xxx 来控制。但是感觉临时用法没有必要暴露给用户？后续版本也去掉了 WITH batch_size

Comment (qiancai) [reply sql-statements/sql-statement-load-data.md:122]: @Frank945946 PTAL

Comment (Frank945946) [reply sql-statements/sql-statement-load-data.md:122]: 不需要暴露出来

**Review by lance6716 — APPROVED** [review]

**Review by Frank945946 — APPROVED** [review]

File: sql-statements/sql-statement-load-data.md (hunk 1 above)
Comment (qiancai) [inline sql-statements/sql-statement-load-data.md:122]:
```suggestion
> - 从 v7.0.0 起，`tidb_dml_batch_size` 对 `LOAD DATA` 语句不再生效，TiDB 将在一个事务中提交所有行。
```

File: sql-statements/sql-statement-load-data.md [hunk 2]
```diff
@@ -117,9 +117,10 @@ TiDB 中的 `LOAD DATA` 语句语法上兼容 MySQL（除字符集选项被解
 
 > **注意：**
 >
-> - 在 TiDB v4.0.0 之前的版本中，`LOAD DATA` 语句每 20000 行进行一次提交。
-> - 从 TiDB v4.0.0 开始一直到 TiDB v6.6.0 的版本，TiDB 默认在一个事务中提交所有行。
-> - 从 TiDB v4.0.0 及以前版本升级后，可能出现 `ERROR 8004 (HY000) at line 1: Transaction is too large, size: 100000058` 错误。要解决该问题，建议调大 `tidb.toml` 文件中的 [`txn-total-size-limit`](/tidb-configuration-file.md#txn-total-size-limit) 值。如果无法增加此限制，还可以将 [`tidb_dml_batch_size`](/system-variables.md#tidb_dml_batch_size) 的值设置为 `20000` 来恢复升级前的行为。注意自 v7.0.0 起，`tidb_dml_batch_size` 对 `LOAD DATA` 语句不再生效。
+> - 在 TiDB v4.0.0 之前的版本中，`LOAD DATA` 语句每 20000 行进行一次提交。该行数不支持更改。
+> - 从 TiDB v4.0.0 开始一直到 TiDB v6.6.0 的版本，TiDB 默认在一个事务中提交所有行。如需 `LOAD DATA` 语句按照每固定的行数进行一次提交，可以设置 [`tidb_dml_batch_size`](/system-variables.md#tidb_dml_batch_size) 为所需的行数。
+> - 从 v7.0.0 起，`tidb_dml_batch_size` 对 `LOAD DATA` 语句不再生效，TiDB 将在一个事务中提交所有行。
```
Comment (qiancai) [inline sql-statements/sql-statement-load-data.md:122]:
```suggestion
> - 从 TiDB v7.0.0 起，`tidb_dml_batch_size` 对 `LOAD DATA` 语句不再生效，TiDB 将在一个事务中提交所有行。
```

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**